contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -166,13 +168,17 @@ In this case, only the `root` component can be accessed in Vue while the rest ar
       template: {
         <button v-on:click="$emit('increased-count', count+1)">Increment item count</button>
       }
-    }
+    })
 
     /* Inside todo-list component */
     template: {
-        v-on:increased-count="updateCount"
+      v-on:increased-count="updateCount"
     }
     ```
+   <box type="tip">
+   
+   When the increment item count button is clicked, the `item` component will emit a custom event named `increased-count` while the `todo-list` component listens to the `increased-count` event and executes it's own `updateCount` method.
+   </box>
```

Comment:
When the ~increment item count~ button is clicked, the `item` component will emit a custom event named `increased-count` while the `todo-list` component listens ~to the `increased-count`~ **for this** event and executes **its** own `updateCount` method.